certbot/main.py

```diff
@@ -275,7 +277,8 @@ def _ask_user_to_confirm_new_names(config, new_domains, certname, old_domains):
                new_domains,
                old_domains))
     obj = zope.component.getUtility(interfaces.IDisplay)
-    if not obj.yesno(msg, "Update cert", "Cancel", default=True):
+    if not obj.yesno(msg, "Update cert", "Cancel",
```

Comment:
This is landing in 0.10.0; I'm not sure whether it should have `force_interactive=True`...